src/commands/appdistribution-distribute.ts

```diff
@@ -0,0 +1,109 @@
+import * as fs from "fs-extra";
+
+import * as Command from "../command";
+import * as utils from "../utils";
+import * as requireAuth from "../requireAuth";
+import { AppDistributionClient } from "../appdistribution/client";
+import { FirebaseError } from "../error";
+import { Distribution } from "../appdistribution/distribution";
+
+function ensureFileExists(file: string, message = ""): void {
+  if (!fs.existsSync(file)) {
+    throw new FirebaseError(`File ${file} does not exist: ${message}`);
+  }
+}
+
+function getAppId(appId: string): string {
+  if (!appId) {
+    throw new FirebaseError("set the --app option to a valid Firebase app id and try again");
+  }
+  return appId;
+}
+
+function getReleaseNotes(releaseNotes: string, releaseNotesFile: string): string {
+  if (releaseNotes) {
+    return releaseNotes;
+  } else if (releaseNotesFile) {
+    ensureFileExists(releaseNotesFile);
+    return fs.readFileSync(releaseNotesFile, "utf8");
+  } else {
```

Comment:
The `else` block can be dropped and the `return "";` can follow the closing bracket of the `else if`